core/src/main/java/org/elasticsearch/action/DocWriteResponse.java

```diff
@@ -128,6 +131,15 @@ public DocWriteResponse(ShardId shardId, String type, String id, long seqNo, lon
         this.result = result;
     }
 
+    public DocWriteResponse(ShardId shardId, String type, String id, String input, long seqNo, long version, Result result) {
+        this.shardId = shardId;
+        this.type = type;
+        this.id = id;
+        this.input = input;     //added
```

Comment:
I think that comment is not necessary, you can delete or try to describe it more specifically.